datasets/kwaliteitsmonitorwonen/dataset.json

```
@@ -0,0 +1,163 @@
+{
+  "id": "kwaliteitsmonitorwonen",
+  "type": "dataset",
+  "title": "Kwaliteitsmonitor Wonen",
+  "description": "Per verblijfsobject de kenmerken van de woningvoorraad van de gemeente Amsterdam.",
+  "version": "1.0.0",
+  "status": "beschikbaar",
+  "theme": [
+    "Wonen"
+  ],
+  "publisher": "OIS",
+  "owner": "Gemeente Amsterdam",
+  "keywords": [
+    "Wonen"
+  ],
+  "crs": "EPSG:28992",
+  "tables": [
+    {
+      "id": "kwaliteitsmonitorwonen",
+	  "title": "Per verblijfsobject de kenmerken van de woningvoorraad van de gemeente Amsterdam.",
+      "type": "table",
+      "version": "1.0.0",
+      "schema": {
+		 "$schema": "http://json-schema.org/draft-07/schema#",
+		 "type": "object",
+		 "additionalProperties": false,
+        "required": [
+          "id",
+          "schema"
+        ],
+        "display": "id",      
+        "properties": {
+          "schema": {
+            "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+          },
+          "id": {
+            "type": "string",
+            "description": "Unieke aanduiding objecttype"
+          },
+          "vboIdentificatie": {
+            "type": "string",
+            "description": "Unieke identificatie van het verblijfsobject"
+          },
+          "vboGeometrie": {
+            "description": "Geometrie van het type POINT van het verblijfsobject",
+            "$ref": "https://geojson.org/schema/Point.json"
+          },
+          "vboOppervlakte": {
+            "type": "integer",
+            "description": "De oppervlakte van het verblijfsobject"
+          },
+          "vboGebruiksdoel": {
+            "type": "string",
+            "description": "Het gebruiksdoel van het verblijfsobject"
+          },
+          "pndIdentificatie": {
+            "type": "integer",
+            "description": "Unieke identificatie van het pand"
+          },
+          "pndGeometrie": {
+            "description": "Geometrie van het type MULTIPOLYGON van het pand",
+            "$ref": "https://geojson.org/schema/Multipolygon.json"
```

Toch nog een foutje: dit moet zijn

```suggestion
            "$ref": "https://geojson.org/schema/MultiPolygon.json"
```

Ik commit 'm wel.